src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -117,63 +118,87 @@ public function __construct(SystemInfoCollector $composerCollector, $debug = fal
+        // In case someone switches from TTL to BUL, make sure we only identify installation as Trial if this is present,
         // as well as TTL packages
         $hasTTLComposerRepo = \in_array('https://updates.ez.no/ttl', $this->composerInfo->repositoryUrls);
 
-        if ($package = $this->getFirstPackage(self::ENTERPISE_PACKAGES)) {
-            $ez->isEnterpise = true;
-            $ez->isTrial = $hasTTLComposerRepo && $package->license === 'TTL-2.0';
-            $ez->name = 'Ibexa DXP';
+        if ($package = $this->getFirstPackage(self::ENTERPRISE_PACKAGES)) {
+            $ibexa->isEnterpise = true;
+            $ibexa->isTrial = $hasTTLComposerRepo && $package->license === 'TTL-2.0';
+            $ibexa->name = 'Ibexa DXP';
         }
 
         if ($package = $this->getFirstPackage(self::COMMERCE_PACKAGES)) {
-            $ez->isCommerce = true;
-            $ez->isTrial = $ez->isTrial || $hasTTLComposerRepo && $package->license === 'TTL-2.0';
-            $ez->name = 'Ibexa Commerce';
+            $ibexa->isCommerce = true;
+            $ibexa->isTrial = $ibexa->isTrial || $hasTTLComposerRepo && $package->license === 'TTL-2.0';
+            $ibexa->name = 'Ibexa Commerce';
         }
 
-        if ($ez->isTrial && isset(self::RELEASES[$eZRelease])) {
-            $months = (new DateTime(self::RELEASES[$eZRelease]))->diff(new DateTime())->m;
-            $ez->isEndOfMaintenance = $months > 3;
+        if ($ibexa->isTrial && isset(self::RELEASES[$ibexaRelease])) {
+            $months = (new DateTime(self::RELEASES[$ibexaRelease]))->diff(new DateTime())->m;
+            $ibexa->isEndOfMaintenance = $months > 3;
             // @todo We need to detect this in a better way, this is temporary until some of the work described in class doc is done.
-            $ez->isEndOfLife = $months > 6;
+            $ibexa->isEndOfLife = $months > 6;
         } else {
-            if (isset(self::EOM[$eZRelease])) {
-                $ez->isEndOfMaintenance = strtotime(self::EOM[$eZRelease]) < time();
+            if (isset(self::EOM[$ibexaRelease])) {
+                $ibexa->isEndOfMaintenance = strtotime(self::EOM[$ibexaRelease]) < time();
             }
 
-            if (isset(self::EOL[$eZRelease])) {
-                if (!$ez->isEnterpise) {
-                    $ez->isEndOfLife = $ez->isEndOfMaintenance;
+            if (isset(self::EOL[$ibexaRelease])) {
+                if (!$ibexa->isEnterpise) {
+                    $ibexa->isEndOfLife = $ibexa->isEndOfMaintenance;
                 } else {
-                    $ez->isEndOfLife = strtotime(self::EOL[$eZRelease]) < time();
+                    $ibexa->isEndOfLife = strtotime(self::EOL[$ibexaRelease]) < time();
                 }
             }
         }
 
-        $ez->stability = $this->getStability();
+        $ibexa->endOfMaintenanceDate = $this->getEOMDate($ibexaRelease);
+        $ibexa->endOfLifeDate = $this->getEOLDate($ibexaRelease);
```

Comment:
Won't be accurate if Trial/OSS with the current wording of properties, see logic above

Reply:
But if prefixed to clarify the sates here are only valid if you have a valid DXP/Enterprise subscription, then it would be fine.

Note: Could probably use this variable in logic above a few places where there are checks against the dates.

Reply:
Or, we can solve this with some information text in the table, something like
End of Maintenance*                     <date>** _(if expired then RED color, if OSS then orange otherwise)_
End of Life*                                     <date>** _(if expired then RED color, if OSS then RED if EOM and orange otherwise)_

* You can read more about Service life for Ibxea DXP with a business license here.
if OSS
** Does not apply to open source versions, for guaranteed maintenance, security fixes before disclosure and much more, upgrade to Ibexa DXP.
elseif Trial
* Only applicable if upgrading to a business license, trial license is firmly limited to 6 months.

Reply:
Too complex, we can rather drop the dates for OSS and Trial and just a `*` explaining it with link to service life if people want to see what they get on that if they upgrade to EE